metadata/demographic/census_places_2020.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Census 2020 Places
+
+## ID
+
+b5f022ca-96c8-4f16-bc47-8bbf35c67db9
+
+## Brief Summary
+
+Incorporated towns and Census Designated Places (CDPs) in Utah from the 2020 Census.
+
+## Summary
+
+Census Places consist of both legally incorporated cities and Census Designated Places (CDPs). This dataset contains boundary information only and does not include demographic data.
+
+## Description
+
+### What is the dataset?
+
+This dataset contains polygon boundaries for cities, towns, metro townships, and census-designated places (CDPs) in Utah. The [US Census Bureau](https://www.census.gov/programs-surveys/bas/information/cdp.html) defines CDPs as "statistical geography representing closely settled, unincorporated communities that are locally recognized and identified by name." Both these unincorporated communities and legally incorporated areas of development are represented in this dataset.
+
+### What is the purpose of the dataset?
+
+The US Census Bureau uses both legally incorporated (cities, towns, etc) and unincorporated places (CDPs) for the once-a-decade count of population and housing. While this dataset does not contain demographic information, it can be joined to other layers to visualize population data.
```

Comment:
```suggestion
The US Census Bureau uses both legally incorporated places (cities, towns, etc) and unincorporated places (CDPs) for the once-a-decade count of population and housing. While this dataset does not contain demographic information, it can be joined to other layers to visualize population data.
```
I feel like `(cities, towns, etc)` separates the `incorporated` and `unincorporated` words enough that we need a separate `place` after each, but I am not an English teacher (though I did stay at a Holiday Inn Express...)